components/dashboard/script.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+
+const vars = {
+    REACT_APP_HOSTNAME : 'Gitpod',
+    REACT_APP_DOCS_URL : 'https://www.gitpod.io/docs/',
+    REACT_APP_SOCIAL_URL : 'https://community.gitpod.io/',
+    REACT_APP_FAVICON_192 : './favicon192.png',
+    REACT_APP_FAVICON_256 : './favicon256.png',
+    REACT_APP_FAVICON_512 : './favicon512.png',
+    REACT_APP_ICON : 'favicon192.png',
+    REACT_APP_BRAND_SVG : 'gitpod.svg'
+};
+
+if(!fs.existsSync('.env')) {
+    let data = "";
+    for(const prop in vars) {
+        data += prop + "=" + vars[prop] + "\n";
+    }
+    fs.writeFileSync('.env', data);
+}
+
+const envObj = {};
+fs.readFileSync('.env', 'utf-8')
+    .split('\n')
+    .forEach(element => {
+        element = element.replace(/\s+/g, '');
+        const temp = element.split('=');
+        envObj[temp[0]] = temp[1];
+    });
+
+let data = fs.readFileSync('./public/manifest.json', 'utf-8')
```

Comment:
<details>
<summary>:warning: <strong>This code uses `let` or `var` to declare one or more variables that are never reassigned.</strong> (click for details)</summary>

A possible improvement would be to declare such a variable with `const` instead.   
  
This change would ensure that it would not be reassigned unintentionally, which could lead to difficulty to comprehend bugs.

**[More details](https://confluence.devfactory.com/x/UA_kEw)**

## The improvement suggestion in this case
```js  
const data = fs.readFileSync('./public/manifest.json', 'utf-8') ...
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence.
</details>

[//]: # (<a id="ced142ed-0d57-434f-b1b0-610a81cc5017" data-finding-id="329518928" href="none" codereview-uuid-reserved-tag data-business-id="05d37d16-d6de-46e5-9cc8-3f767198a252" data-analyzer="js" />)